stf/src/sgx.rs

```diff
@@ -497,6 +485,47 @@ fn increment_nonce(account: &AccountId) {
 	}
 }
 
+#[cfg(feature = "test_accounts")]
+fn endow_alice_account(state: &mut SgxExternalities) {
+	state.execute_with(|| {
+		sp_io::storage::set(&storage_value_key("Sudo", "Key"), &ALICE_ENCODED);
```

Comment:
This is more than endowing. It set the sudo account.

Reply:
So I just call it `setup_alice _account` ?

Reply:
I think I would separate the logic as I wrote below.

For me, endowing the accounts is a different thing than setting sudo. It is just coincidently Alice in our case and might change.